src/Microsoft.DotNet.XHarness.iOS.Shared/Listeners/TunnelBore.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.DotNet.XHarness.iOS.Shared.Execution;
+using Microsoft.DotNet.XHarness.iOS.Shared.Logging;
+
+namespace Microsoft.DotNet.XHarness.iOS.Shared.Listeners {
+
+	// manages the tunnels that are used to communicate with the devices. We want to create a single tunnel per device
+	// since we can only run one app per device, this should not be a problem.
+	//
+	// definition: 
+	// A tunnel boring machine, also known as a "mole", is a machine used to excavate tunnels with a circular cross section
+	// through a variety of soil and rock strata. They may also be used for microtunneling. 
```

Comment:
nit: It would also be better do do the whole `/// <summary>` sharade with docs

+1 for the definition 😀